packetbeat/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Beats dockerfile used for testing
+FROM ruflin/beat-tester
```

Comment:
I actually went to docker hub to look before posting :) , but it didn't have the Dockerfile listed.

I know why the line is at the end now, but it's not obvious that it is because of caching. It's quite subtle that most of this Dockerfile is the same as the libbeat Dockerfile and therefore they share the same layers. I'm sure with all of your docker and docker-compose wizardry that the duplication can be removed in the future.